src/en/components/card/design.md

```diff
@@ -0,0 +1,52 @@
+---
+title: card - Components
+layout: "layouts/component-documentation.njk"
+translationKey: "cardDesign"
+tags: ['cardEN', 'design']
+date: "git Last Modified"
+---
+
+## Card anatomy
+
+<ol class="anatomy-list">
+  <li>The <strong>container</strong> holds all of a card’s elements.</li>
+  <li>The <strong>image</strong> is a 1:1 or a 16:9 visual that can be included in the card.</li>
+  <li>The <strong>tag</strong> is a piece of supporting information that can be included to describe the content type.</li>
+  <li>The <strong>card title</strong> is a short headline on the subject of the card content and links to further information.</li>
+  <li>The <strong>description</strong> has supporting text for the card title and includes a few lines of summary on the card’s topic or a progressive list of details related to the same subject, like a list of event details.</li>
+  <li>The <strong>card footer</strong> can contain either an action, when there’s a button, or metadata. Metadata can include a date, location, author, or other relevant information.</li>
+</ol>
+
+<img class="b-sm b-default p-400" src="/images/en/components/anatomy/gcds-card-anatomy.svg" alt="An image showing a card's anatomy"/>
```

Comment:
Could you please update the anatomy image to use this image instead? Since we are listing the title as `card title`, we should be using similar wording in the anatomy image to be consistent.

https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/branch/LXmH6TsKGdG1lEVO7O1Qv6/Canadian-Digital-Service%E2%80%A8---GC-Design-System?type=design&node-id=7628-11006&mode=design&t=hpk1GYNMrHVqdHne-0